bindings/python/pyiree/rt/function_abi.cc

```diff
@@ -12,19 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "bindings/python/pyiree/rt/function_abi.h"
+#include "function_abi.h"
 
 #include "absl/memory/memory.h"
 #include "absl/strings/str_cat.h"
 #include "absl/types/span.h"
-#include "bindings/python/pyiree/common/status_utils.h"
-#include "bindings/python/pyiree/rt/hal.h"
-#include "bindings/python/pyiree/rt/vm.h"
+#include "hal.h"
```

Comment:
it's pretty unfortunate that the local headers all get mixed in here by clang-format as it's really messy - can we still use the project-rooted paths? or, pyiree-rooted paths? (pyiree/rt/hal.h)

Reply:
That's dumb and super annoying -- especially after I specifically separated them all.

I'd like to be able to produce a source wheel for this stuff, and for that, the local files should be self contained. Maybe I'll do some comment thing to separate them. Also open to prefixing or relative access with "./". The intent is that this directory may be an island that can compile independently against the public IREE API (it will be a bit before we are back to that).